### PR TITLE
feat(http): implement stream support

### PR DESCRIPTION
**Pull request by TheNightmareX**
### Summary of changes

feat(http): implement stream support

### Relinquishment of Rights

Please mark following checkbox to confirm that you relinquish all rights of your changes:

- [x] I waive and relinquish all rights regarding this changes (including code, text, and images) to Deepkit UG (limited), Germany. This changes (including code, text, and images) are under MIT license without name attribution, copyright notice, and permission notice requirement.

### CODE REVIEW

File: packages/http/tests/router.spec.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { HttpBody, HttpBodyValidation, HttpQueries, HttpQuery, HttpRegExp, HttpR
 import { getClassName, sleep } from '@deepkit/core';
 import { createHttpKernel } from './utils';
 import { Group, MinLength } from '@deepkit/type';
+import { Readable } from 'stream';
+import { createReadStream } from 'fs';
```
Comment (marcj) [inline packages/http/tests/router.spec.ts:11]: not used

File: packages/http/src/http.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ import { createWorkflow, WorkflowEvent } from '@deepkit/workflow';
 import type { ElementStruct, render } from '@deepkit/template';
 import { FrameCategory, Stopwatch } from '@deepkit/stopwatch';
 import { getSerializeFunction, hasTypeInformation, ReflectionKind, resolveReceiveType, SerializationError, serialize, serializer, ValidationError } from '@deepkit/type';
+import stream from 'stream';
```
Comment (marcj) [inline packages/http/src/http.ts:22]: could we decouple this from nodejs? So that we also support Deno/etc

**Comment by marcj** [comment]
Cool, Codecov works again💪

File: packages/http/src/http.ts (hunk 2 above)
Comment (marcus-sa) [reply packages/http/src/http.ts:22]: @TheNightmareX exactly. I've already created an issue for that https://github.com/deepkit/deepkit-framework/issues/236

**Comment by marcj** [comment]
Thanks! merged in https://github.com/deepkit/deepkit-framework/commit/8639d896730d816917f05e13b0ef897604227745. Unfortunately I failed pulling it correctly so it lost author information, sorry about that!

**Pull request closed without merging**